src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,16 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  comments: Comment[];
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <div className="CommentList">
```

Comment:
use semantic tags for a list (i.e `ul`, `li`)
P.S. Don't forget to fix the styles